megamek/src/megamek/common/AmmoType.java

```diff
@@ -13432,13 +13437,15 @@ public AmmoType createMunitionType(AmmoType base) {
                     || (munition.getAmmoType() == AmmoType.T_NLRM))
                     && (munition.getMunitionType() == AmmoType.M_AX_HEAD)) {
                 cost *= 0.5;
+                bv *= 1;
```

Comment:
multiplying by 1 doesn't do anything?

Reply:
Reminder that the BV has been checked. Beats have a blank line with no idea that it's been compared to source.

Reply:
Can we comment it out instead then?